orchagent/portsorch.cpp

```diff
@@ -1498,20 +1521,26 @@ bool PortsOrch::initPort(const string &alias, const set<int> &lane_set)
                 /* Add port to port list */
                 m_portList[alias] = p;
                 m_port_ref_count[alias] = 0;
-                /* Add port name map to counter table */
-                FieldValueTuple tuple(p.m_alias, sai_serialize_object_id(p.m_port_id));
-                vector<FieldValueTuple> fields;
-                fields.push_back(tuple);
-                m_counterTable->set("", fields);
-
+                {
```

Comment:
Why are these brackets needed?

Reply:
Just limiting the scope of `fields` variable.